.github/workflows/release.yaml

```diff
@@ -0,0 +1,125 @@
+
+      - name: Download and extract compiled LLVM release
+        if: steps.cache-llvm.outputs.cache-hit != 'true'
+        env:
+          LLVM: ${{ env[format('LLVM_{0}', matrix.arch)] }}
+        run: |
+          curl -L https://github.com/llvm/llvm-project/releases/download/llvmorg-15.0.0/${{ env.LLVM }}.tar.xz -O
+          tar xvf ${{ env.LLVM }}.tar.xz
+
+      - name: Checkout bpftool code
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f
+        with:
+          submodules: recursive
+          path: 'bpftool'
+
+      - name: Build static bpftool natively for amd64
+        if: matrix.arch == 'amd64'
+        working-directory: 'bpftool'
+        env:
+          LLVM: ${{ env[format('LLVM_{0}', matrix.arch)] }}
+        run: make -j $(nproc) -C src V=1 \ 
+          EXTRA_CFLAGS=--static \
+          LLVM_CONFIG="${GITHUB_WORKSPACE}/${{ env.LLVM }}/bin/llvm-config" \
+          LLVM_STRIP="${GITHUB_WORKSPACE}/${{ env.LLVM }}/bin/llvm-strip"
+
+      - name: Strip binary (amd64)
+        if: matrix.arch == 'amd64'
+        run: strip bpftool/src/bpftool
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18
+        if: matrix.arch == 'arm64'
+        with:
+          platforms: arm64
+
+      # The emulated build leverage Docker and Ubuntu 22.04 container image
+      # distribution to have all the needed arm64 packages.
+      - name: Build static bpftool for arm64 with emulation
+        if: matrix.arch == 'arm64'
+        env:
+          LLVM: ${{ env[format('LLVM_{0}', matrix.arch)] }}
+        run:  |
+          docker run --platform linux/arm64 --rm -v $(pwd):/build ubuntu:22.04 \
+          bash -c "apt-get update && \
+                   apt-get install -y make pkg-config libelf-dev libcap-dev clang gcc llvm binutils && \
+                   cd build/bpftool && \
+                   LLVM_CONFIG=../../${{ env.LLVM }}/bin/llvm-config EXTRA_CFLAGS=--static make -j V=1 -C src && \
+                   strip src/bpftool"
+
+      - name: Test bpftool binary
+        working-directory: 'bpftool/src'
+        run: |
+          file ./bpftool
+          ./bpftool 2>&1 | grep -q Usage
+          ./bpftool -p version | \
+              tee /dev/stderr | \
+              jq --exit-status ".features | .llvm"
+          ldd ./bpftool 2>&1 | \
+              tee /dev/stderr | \
+              grep -q 'not a dynamic executable'
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce
+        with:
+          name: ${{ format('bpftool_{0}', matrix.arch) }}
+          path: bpftool/src/bpftool
+
+  draft-release:
+    name: Create a draft release
+    runs-on: ubuntu-22.04
+    needs: build
+    permissions:
+      contents: write
+    steps:
+      - name: Download artifacts from build
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a
```

Comment:
![image](https://user-images.githubusercontent.com/17001771/226426455-ae72fa9b-04d6-4d68-accd-b0b7aaf42b57.png)

But it looks like you're using v3.0.2 already? :thinking: 

By the way I've seen other repositories adding the version in comments:

```
        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
```

Could be nice if you don't mind, but don't bother if you don't feel like it.

Reply:
Ah, that would make sense, sorry